src/apollo/queries.js

```diff
@@ -13,10 +13,10 @@ export const GET_RUNS = gql`
   }
 `;
 
-/** query for details metadata component */
-export const GET_RUN_METADATA = gql`
-  query getRunMetadata($runIds: [ID!]!) {
-    runMetadata(runIds: $runIds) {
+/** query for runMetadata and runDataset components */
+export const GET_RUN_DATA = gql`
+  query getRunData($runIds: [ID!]!, $showDiff: Boolean) {
+    metadata: runMetadata(runIds: $runIds) {
```

Comment:
There's no problem with how you have it here, but you don't actually need to alias `runMetadata` as `metadata`. The only reason it's needed for `plots` etc. is because otherwise there would be multiple fields named `runTrackingData`, which would clash. There's only one `runMetadata` here so there would not be a clash in the first place.

Reply:
hi @AntonyMilneQB -- it doesn't work. because when i go to fetch queries in experiment wrapper, i am not able to say what i want to extract. 
see - https://github.com/kedro-org/kedro-viz/blob/b1f974f198fdfc74d77c57542f70712513a82f13/src/components/experiment-wrapper/experiment-wrapper.js#L31